bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -201,6 +205,19 @@ func Run(opt *options.ServerOption) error {
 		return err
 	}
 
+	log.Infof("create rolebinding kubeflow-admin for role permission")
+	user, err := exec.Command("gcloud", "config", "get-value", "account").Output()
```

Comment:
We shouldn't be running gcloud commands unless we are using GKE. You can use the isGke function to check.